examples/dut_communication_using_semi_device_control_nidigital.py

```diff
@@ -44,18 +44,20 @@
 
     # Attach to the existing 657x gRPC session.
     # Get the resource name of the instrument session for NI 657x device.
-    resource_name = semi_device_control.get_instrument_session("NI 657x").ResourceName
+    grpc_session_options = semi_device_control.get_grpc_session_options("NI 657x")
```

Comment:
Why `NI 657x` is hardcoded? What's the input parameter needed for `get_grpc_session_options`? Do we have to pass in the interface name?

Reply:
Yes, interface name is a required parameter for get_grpc_session_options()

Reply:
Can we create a variable with this hardcoded value and use?